allure-framework-modules/mrchecker-webapi-module/src/main/java/com/capgemini/ntc/webapi/core/base/runtime/RuntimeParameters.java

```diff
@@ -0,0 +1,54 @@
+package com.capgemini.ntc.webapi.core.base.runtime;
+
+import com.capgemini.ntc.test.core.base.runtime.RuntimeParametersI;
+
+/**
+ * This class stores various system properties These parameters are accessible while test case executes Example :
+ * PARAM_3("param_3", "1410") IS: mvn test -Dparam_3=1525 -Dtest=MyTestClass
+ * 
+ * @author LUSTEFAN
+ */
+public enum RuntimeParameters implements RuntimeParametersI {
```

Comment:
When you do not use any RuntimeParameters, then please remove this class.
If in future this is needed, then you can always copy this from "module-example"